### ISSUE TITLE
implement panel as proper gobject

### ISSUE THREAD

**Issue by aanatoly** [aanatoly/fbpanel#15]
Currently it's a `GtkWindow` with connected signals that try to overtake normal flow. It become messy.

**Comment by RichJack** [comment]
Is this a good opportunity to port to GTK+3?

**Comment by aanatoly** [comment]
Nope. gtk3 is not in a scope. at least as long as gtk2 does the job.
